DashAI/back/dataloaders/classes/dataloader.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
+
+from datasets import ClassLabel
+
+from DashAI.back.config_object import ConfigObject
+
+
+class DataLoader(ConfigObject):
+    """
+    Abstract class with base methods for all data loaders
+    """
+
+    @abstractmethod
+    def load_data(self, dataset_path, url=None):
+        pass
+
+    @abstractmethod
+    def set_task_format(self, task):
+        pass
```

Comment:
That method is not necessary anymore, so I removed it. because now the validation that the dataset has the format for the task, is done by the task itself.